pkg/model/segment.go

```diff
@@ -30,10 +30,11 @@ type SegmentMediaData struct {
 	Video    []*SegmentMediadataVideo `json:"video"`
 	Audio    []*SegmentMediadataAudio `json:"audio"`
 	Duration int64                    `json:"duration"`
+	SizeB    int                      `json:"sizeB"`
```

Comment:
Probably could live with just calling this `size` for consistency unless that's reserved somewhere or something